### PR TITLE
92073 - update gradle

### PR DESCRIPTION
**Pull request by skiesewetter-intershop**

### CODE REVIEW

File: README.asciidoc
Comment (DevDavido) [inline README.asciidoc:1]: After removing the deprecated `ProjectServerDirs` object, there are remaining references in this README, see `ProjectServerDirs`.

Comment (DevDavido) [inline README.asciidoc:1]: Please update README for section `Version ??? requires Gradle 7.5.1!`

File: build.gradle.kts [hunk 1]
```diff
@@ -38,29 +37,23 @@ plugins {
...
+    `jvm-test-suite`
 
     // plugin for documentation
     id("org.asciidoctor.jvm.convert") version "3.3.2"
 
     // documentation
-    id("org.jetbrains.dokka") version "1.5.0"
-
-    // code analysis for kotlin
-    id("io.gitlab.arturbosch.detekt") version "1.18.0"
```
Comment (DevDavido) [inline build.gradle.kts:51]: Please remove the `detekt.yml` configuration file too

File: build.gradle.kts [hunk 2]
```diff
@@ -317,12 +324,8 @@ signing {
 
 dependencies {
     implementation(gradleApi())
-    implementation(localGroovy())
+    implementation(gradleKotlinDsl())
 
-    compileOnly("org.apache.ant:ant:1.10.12")
+    compileOnly("org.apache.ant:ant:1.10.14")
```
Comment (DevDavido) [inline build.gradle.kts:332]: Is there any usage of the `ant` dependency or can this be removed?

File: src/main/kotlin/com/intershop/gradle/icm/extension/DevelopmentConfiguration.kt [hunk 3]
```diff
@@ -146,23 +123,6 @@ open class DevelopmentConfiguration
         }
     }
 
-    /**
```
Comment (DevDavido) [inline src/main/kotlin/com/intershop/gradle/icm/extension/DevelopmentConfiguration.kt:149]: Please remove all the license configuration related documentation from README too.

**Review by DevDavido — CHANGES_REQUESTED** [review]

**Review by DevDavido — APPROVED** [review]

**Review by cortlepp-intershop — APPROVED** [review]

**Pull request merged**